set.mm

```diff
@@ -74725,6 +74794,60 @@ of related elements (Contributed by Alexander van der Vekens,
       EIXEWE $.
   $}
 
+  ${
+    $d A a b x y z $.  $d B a b x y z $.  $d F a b c x y z $.
+    $d G a b c x y z $.  $d X a b c z $.  $d Y a b c z $.
+    fvproj.h $e |- H = ( x e. A , y e. B |-> <. ( F ` x ) , ( G ` y ) >. ) $.
+    ${
+      fvproj.x $e |- ( ph -> X e. A ) $.
+      fvproj.y $e |- ( ph -> Y e. B ) $.
+      $( Value of a function on pairs, given two projections ` F ` and ` G ` .
```

Comment:
```suggestion
      $( Value of a function on ordered pairs with values expressed as ordered pairs. Note that ` F ` and ` G ` are the projections of ` H ` to the first and second coordinate respectively.
```
Justification:
* it is important to say that the values are ordered pairs and are given explicitly as ordered pairs
* I avoided "given two projections", because the hypothesis does not literally say "here are two projections". We humans simply note, outside the formal systems, that these are the projections.

Reply:
I can change this, but I would like to hear @tirix `s opinion because it is from his mathbox.

Reply:
To me, the point of the comment is exactly to provide the information that we, as humans, need to better understand what the theorem means, and what it's used for. I usually prefer that to simply "reading alout" what the math formula says.

That's why I mentioned "projection" prominently in the description. 

It's like saying "The X operation commutes" rather than "the result of the X operation is unchanged when operands are swapped".

I'm fine with any rewriting which keeps this spirit.

Reply:
Your example with commutation is not the same. Here, if I began by reading your comment, I would formalize it as
```
$e |- H Fn ( A X. B ) $.
$e |- F = ( 1st o. H ) $.
$e |- G = ( 2nd o. H ) $.
...
```
In other words, in order to read `fvproj` as the formalization of your comment, we need the result
```
$e |- H = ( x e. A , y e. B |-> <. ( F ` x ) , ( G ` y ) >. ) $.
$e |- F Fn A $.
$e |- G Fn B $.
$p |- ( F = ( 1st o. H ) /\ G = ( 2nd o. H ) ) $= ? $.
```

Reply:
Yes, sure!